src/ddmd/builtin.d

```diff
@@ -157,6 +160,306 @@ extern (C++) Expression eval_bsr(Loc loc, FuncDeclaration fd, Expressions* argum
     return new IntegerExp(loc, k, Type.tint32);
 }
 
+version(IN_LLVM)
+{
+
+private Type getTypeOfOverloadedIntrinsic(FuncDeclaration fd)
```

Comment:
Maybe same as per comment on pragmas.  Need a way to extend front-end built-ins for each compiler (although gdc just ignores this module).